github/data_source_github_ip_ranges.go

```diff
@@ -144,6 +169,16 @@ func dataSourceGithubIpRangesRead(d *schema.ResourceData, meta interface{}) erro
 		return err
 	}
 
+	cidrWebIpv4, cidrWebIpv6, err := splitIpv4Ipv6Cidrs(&api.Api)
```

Comment:
The properties present on the `ApiMeta` struct are defined [here](https://github.com/google/go-github/blob/master/github/misc.go#L142) and don't include `Api` or `Web`. As best I can tell looking at the [file history](https://github.com/google/go-github/commits/master/github/misc.go), they haven't included that information in the past, either. 

Perhaps there's another way of getting those CIDRs?

Reply:
@kfcampbell You're right. I've opened a PR in the Go provider to support exposing these `API` and `Web` blocks. Once that's in and a part of a release I'll revisit this.

Reply:
Change was merged, just waiting on https://github.com/integrations/terraform-provider-github/pull/1258 and I'll get any changes in and we can try this again 😄

Reply:
Approved/merged!

Reply:
@kfcampbell updates made, thanks for the reminder!